runtime/nls/shrc/j9shr.nls

```diff
@@ -6802,6 +6802,13 @@ J9NLS_SHRC_SHRINIT_HELPTEXT_CREATE_LAYER.system_action=
 J9NLS_SHRC_SHRINIT_HELPTEXT_CREATE_LAYER.user_response=
 # END NON-TRANSLATABLE
 
+J9NLS_SHRC_SHRINIT_HELPTEXT_PORTABLE=Enable the shared cache to be portable.
```

Comment:
@hangshao0 Not sure if I should add this or this is someone else's job. It looks like it needs to be in multiple languages.

Reply:
Adding the current message in this file is sufficient now. Our translators will translator this message into different languages and add them in the corresponding `.nls` files.

Reply:
Please add this message at the end of this file.